src/hook/pauseResume.js

```diff
@@ -0,0 +1,318 @@
+/**
+* Copyright (c) 2015 NAVER Corp.
+* egjs projects are licensed under the MIT license
+*/
+eg.module("pauseResume", ["jQuery"], function($) {
+	"use strict";
+
+	var animateFn = $.fn.animate;
+	var stopFn = $.fn.stop;
+	var uuid = 1;
+	var getStyles;
+
+	if (window.getComputedStyle) {
```

Comment:
- When use animate in IE8([link](https://msdn.microsoft.com/en-us/library/ms535231(v=vs.85).aspx)) that Including dash style(like `padding-left`) didn`t working.
- It have performance issue using `getComputedStyle`. But If You don`t change DOM It haven`t issue.